### PR TITLE
Added WtD Prague 2017 writeup by Catherine Heath. [WIP/Do Not Merge]

### PR DESCRIPTION
**Pull request by CrispyCB**

### CODE REVIEW

File: docs/conf/na/2016/index.rst [hunk 1]
```diff
@@ -274,6 +274,7 @@ Write-Ups From the Community
 -  `Write the Docs 2016 <https://www.davidbaumgold.com/blog/2016/05/wtd-na-2016/>`_ by David Baumgold
 -  `Write the Docs 2016 Recap <https://circleci.com/blog/write-the-docs-2016-recap/>`_ by Ricardo N Feliciano
 -  `Write the Docs NA 2016 wrapup <https://ffeathers.wordpress.com/2016/05/25/write-the-docs-na-2016-wrapup/>`_ by Sarah Maddox
+-  `Write the Docs EU 2017 writeup <https://www.knowledgeowl.com/home/write-the-docs-prague-conference-2017>`_ by Catherine Heath
```
Comment (ericholscher) [inline docs/conf/na/2016/index.rst:277]: This is a 2017 EU write up --- it should go in the EU 2017 conference page! :)

**Review by ericholscher — COMMENTED** [review]
Thanks for the contribution -- I noted one change below.

**Comment by plaindocs** [comment]
Thanks for fixing this @CrispyCB ! Great stuff!

You probably need to specify the 2017 conf templates at the start of the file, look at the other `2017/eu` pages to see how to do that. 

And because it is a new page, we'll need to add it to the menu for Desktop and Mobile. I can help you with that if you ping me when you're done with other changes.

**Comment by plaindocs** [comment]
hey folks, do you need any help pushing this through?

**Comment by plaindocs** [comment]
Closing this, feel free to reopen.

**Pull request closed without merging**